plugins/ocm/README.md

```diff
@@ -1,104 +1,114 @@
 # Open Cluster Management plugin for Backstage
 
-This plugin integrates your Backstage instance with Open Cluster Management's MultiClusterHub and MultiCluster Engine.
+The Open Cluster Management (OCM) plugin integrates your Backstage instance with the `MultiClusterHub` and `MultiCluster` engines of Open Cluster Management.
```

Comment:
```suggestion
The Open Cluster Management (OCM) plugin integrates your Backstage instance with the `MultiClusterHub` and `MultiCluster` engines of OCM.
```

can be OCM in the second instance since it's defined earlier in the sentence?

Reply:
Done